themes/sandal/less/sandal.less

```diff
@@ -240,7 +240,8 @@ a:hover .login-dropdown-caret {
 
   .more-btn, .less-btn {
     border: 0;
-    padding: 0;
+    border-bottom: 1px solid #ddd;
```

Comment:
Works, but should use the variable for border color:
`    border-bottom: 1px solid @list-group-border;`

Reply:
Thank you -- I suspected I should use a variable, but for whatever reason I had trouble finding it. I'll fix this momentarily.

Reply:
Fixed!